jina/types/document/__init__.py

```diff
@@ -42,6 +43,37 @@
 
 _all_mime_types = set(mimetypes.types_map.values())
 
+scipy_installed = False
+tensorflow_installed = False
+pytorch_installed = False
+
+with ImportExtensions(
+    required=False,
+    pkg_name='scipy',
+    help_text=f'can not import scipy: pip install scipy ',
```

Comment:
It is good to have the check here if the package is installed.
Is the `help_text` required? Or is it something the `ImportExtensions` module does not provide itself. 
In case `ImportExtensions` is not capable of giving the right install requirements, we could create a follow-up ticket to add the missing functionality.